table_connector.py

```diff
@@ -257,3 +258,14 @@ def get_cleaned_fully_merged_messages():
     print 'Printing first row of address book with full name column (merged first name/last name/company):'
     display(address_book_df.head(1))
     return fully_merged_messages_df, address_book_df
+
+if __name__=="__main__":
+    messages, addresses = get_cleaned_fully_merged_messages()
+    if len(sys.arv)>1:
+        file_path = sys.argv[1]
+        messages.to_csv(file_path+"messages.csv")
+        addresses.to_csv(file_path+"addresses.csv")
+    else:
+        print messages
```

Comment:
If we wanted that functionality, we could do `print messages.to_csv()`, but I think it's actually prettier to just print it. 

The only argument for printing a csv is if it were going to be redirected from the command-line, for consistency. I was envisioning that I'd either inspect from the command line or save to a file. But I could go either way.